modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -213,8 +242,14 @@ private void _addObjectRelationshipSchema(
 				_bundleContext, getExternalDTOClassName(objectDefinition),
 				_openAPIResource);
 		}
+		else if (GetterUtil.getBoolean(
+					PropsUtil.get("feature.flag.LPS-168462"))) {
+
+			sourceSchemas = _objectEntryOpenAPIResource.getSchemas(
+				objectDefinition);
```

Comment:
Here is where I would use the ObjectEntryOpenAPIResourceProvider to get the objectEntryOpenAPIResouce associated with the objectDefinition.